util/compression_test.cc

```diff
@@ -220,6 +477,30 @@ TEST_F(DBAutoSkip, DISABLED_AutoSkipCompressionManager) {
     ASSERT_OK(Flush());
   }
 }
+TEST_F(DBCPUIOPredictor, CUPIOAwareCompressorManager) {
```

Comment:
"DBCPUIOPredictor" is not a good name since CPUIOPredictor is more on compression side but not the whole db.